data/kata-collect-data.sh.in

```diff
@@ -543,6 +552,38 @@ release_device()
 	losetup -d "$device"
 }
 
+show_throttler_details()
+{
+	heading "KSM throttler"
+
+	subheading "version"
+
+	local throttlers
+	local throttler
+
+	throttlers=$(find /usr/libexec /usr/lib* -type f |\
```

Comment:
just for ref, `find` has a `-regex` argument that might do most of this for you.

Reply:
Good point. I think it would be tricky to get a regex to handle the exclude and the include though so probably clearer as-is.